### DISCUSSION TITLE
[Feature Request] Don't filter the results, only detect

### DISCUSSION THREAD

**Discussion by Sh4d0wHunt3rX** [projectdiscovery/cdncheck#443] [Ideas]
For example, I have 10 IPs in a text file, 5 are normal IPs without any cdn or anything. 

I want cdncheck checkes this file and add the info ( -resp ) in front of each IP that detects, but not delete the others. So, no filtering, only detecting.

**Comment by GeorginaReeder** [comment]
Thanks for opening this feature request @amiremami , we'll take a look into it!
